py-polars/polars/_utils/deprecation.py

```diff
@@ -77,31 +77,6 @@ def wrapper(*args: P.args, **kwargs: P.kwargs) -> T:
     return decorate
 
 
-def deprecate_streaming_parameter() -> Callable[[Callable[P, T]], Callable[P, T]]:
-    """Decorator to mark `streaming` argument as deprecated due to being renamed."""
-
-    def decorate(function: Callable[P, T]) -> Callable[P, T]:
-        @wraps(function)
-        def wrapper(*args: P.args, **kwargs: P.kwargs) -> T:
-            if "streaming" in kwargs:
-                issue_deprecation_warning(
-                    "the `streaming` parameter was deprecated in 1.25.0; use `engine` instead."
-                )
-                if kwargs["streaming"]:
-                    kwargs["engine"] = "old-streaming"
-                elif "engine" not in kwargs:
-                    kwargs["engine"] = "in-memory"
-
-                del kwargs["streaming"]
-
-            return function(*args, **kwargs)
-
-        wrapper.__signature__ = inspect.signature(function)  # type: ignore[attr-defined]
-        return wrapper
-
-    return decorate
-
-
```

Comment:
This I am not 100% sure about. Should we not dispatch `streaming=True` to `engine="streaming"` until 2.0?